### PR TITLE
CLB gotcha: 503s even when instances are available

### PR DESCRIPTION
**Pull request by pnc**
This bit me once, and it's bit friends before!

There's probably a more concise way to explain it. I haven't used ALBs, so I'm not sure if they exhibit this same behavior.

### CODE REVIEW

**Comment by ThanosBaskous** [comment]
Great addition, @pnc! This has burned me too! LGTM.

@jlevy would you mind checking this out for final review/merge?

**Comment by pnc** [comment]
If there's a preferred style for acronyms (expand on first definition, <abbr> tag, something else) for "AZ", please let me know and I'll update. The acronyms can be a bit overwhelming, for sure!

File: README.md [hunk 1]
```diff
@@ -871,6 +871,7 @@ CLB
 ### CLB Gotchas and Limitations
 
 -	In general, CLBs are not as “smart” as some load balancers, and don’t have fancy features or fine-grained control a traditional hardware load balancer would offer. For most common cases involving sessionless apps or cookie-based sessions over HTTP, or SSL termination, they work well.
+-       By default, CLBs will refuse to route traffic from a load balancer in one Availability Zone to a backend instance in another. This [will cause 503s](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/ts-elb-error-message.html#ts-elb-errorcodes-http503) if the last instance in an AZ becomes unavailable, even if there are healthy instances in other zones. If you’re running fewer than two backend instances per AZ, you almost certainly want to [enable cross-zone load balancing](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-disable-crosszone-lb.html#enable-cross-zone).
```
Comment (jlevy) [inline README.md]: This is an important enough "gotcha" let's put a 🔸 at the front?

**Comment by jlevy** [comment]
Great addition! It's an important one, for sure. Just the one further small request.

**Comment by ThanosBaskous** [comment]
I think that for things that aren't canonically but are conveniently abbreviated (Availability Zone, AZ), first use in a "block" (paragraph, section, whatever) should be expanded, and subsequent uses can be abbreviated.

For things that are almost always abbreviated (ELB, RDS), the full name should appear at least once at the beginning of that component's section in the larger document, but the abbreviation should be used throughout the rest of the document.

File: README.md (hunk 1 above)
Comment (ThanosBaskous) [inline README.md]: 👍 on adding 🔸

**Comment by jlevy** [comment]
@ThanosBaskous I guess we need to put that into the CONTRIBUTING.md too.

File: README.md [hunk 2]
```diff
@@ -871,7 +871,7 @@ CLB
...
+-       🔸 By default, CLBs will refuse to route traffic from a load balancer in one Availability Zone (AZ) to a backend instance in another. This [will cause 503s](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/ts-elb-error-message.html#ts-elb-errorcodes-http503) if the last instance in an AZ becomes unavailable, even if there are healthy instances in other zones. If you’re running fewer than two backend instances per AZ, you almost certainly want to [enable cross-zone load balancing](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-disable-crosszone-lb.html#enable-cross-zone).
```
Comment (ThanosBaskous) [inline README.md]: Nit: We haven't been putting spaces after the markers.

**Comment by pnc** [comment]
Nice catch! Did it in the online editor and didn't see the extra space. I trust you'll squash out the noise during merge, or I can if desired.

**Comment by ThanosBaskous** [comment]
👍 Thanks @pnc!

**Pull request merged**